terraform/github/teams.tf

```diff
@@ -215,6 +222,13 @@ locals {
       ]
     }
 
+    asdf-kp = {
```

Comment:
kpack is more clear, is there a reason to keep the name to kp?

Reply:
You're right - "kpack" is more clear. `kp` is the name of the binary that is installed. Updated to `asdf-kpack-cli` which is the name of the `brew` install on their website, and the name of the source repository: https://github.com/buildpacks-community/kpack-cli